accepted/0000-replace-dwr.md

```diff
@@ -0,0 +1,74 @@
+```
+Cookie “DWRSESSIONID” will be soon rejected because it has the “SameSite” attribute set to “None” or an invalid value, without the “secure” attribute. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite
+```
+
+## What use cases does it support?
+DWR is a library that hide the details of request/response handling during AJAX interactions, it allows JavaScript code to use Java methods running on the backend just as if they were pure JavaScript code. Currently, it supports some features in Suma and the points where it is used are mapped as follows.
+
+* `engine.js` is the core js file from DWR, it is imported in [`layout_head.jsp`](https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/decorators/layout_head.jsp#L67) and referenced in [`template.js`](https://github.com/uyuni-project/uyuni/blob/master/web/html/src/styleguide/template.js#L31), so loaded in all pages.
+* `util.js` is a js library that comes with DWR and contains utility functions that can be used to update HTML using js data. It is imported in [`layout_head.jsp`](https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/decorators/layout_head.jsp#L68) and referenced in [`template.js`](https://github.com/uyuni-project/uyuni/blob/master/web/html/src/styleguide/template.js#L32), so also loaded in all pages. Its functions seem to be used only [here](https://github.com/uyuni-project/uyuni/blob/master/web/html/javascript/spacewalk-checkall.js#L193) and [here](https://github.com/uyuni-project/uyuni/blob/master/web/html/src/manager/audit/cveaudit/cveaudit.tsx#L119).
+* **Dynamically generated JS files** - the following js files are generated dynamically by DWR, exposing some methods of Java classes as pure js functions that call the real code using Ajax requests. The classes and methods are defined in [`dwr.xml`](https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/dwr.xml) file: 
+  * `DWRItemSelector`: exposes the method [`select`](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/frontend/taglibs/DWRItemSelector.java#L52) of the DWRItemSelector java class. It is loaded in [`layout_head.jsp`](https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/decorators/layout_head.jsp#L69) and referenced in [template.js](https://github.com/uyuni-project/uyuni/blob/master/web/html/src/styleguide/template.js#L33), so loaded in all pages. The function is currently used in all the pages using the `rl:selectablecolumn` custom tag defined [here](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/frontend/taglibs/rhn-list.tld#L242), including SSM. **This seems to be the most risky component, since it is used across several pages**.
```

Comment:
This one is used for the SSM Count. For sure it is used in a lot of pages, but as long as we find a proper way to implement it using API endpoints it should be easy to propagate to React pages at least.